packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -109,10 +109,6 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
       log: [{ cid: commitData.cid, type: CommitType.GENESIS }],
     }
 
-    if (state.metadata.schema) {
-      await this._schemaValidator.validateSchema(context.api, state.content, state.metadata.schema)
-    }
```

Comment:
Maybe just comment out this code so that we can put it back once we add the model schema validation?

Reply:
Done.